### DISCUSSION TITLE
renovate-config-validators fails on experimental-manager

### DISCUSSION THREAD

**Discussion by AdrienLemaire** [renovatebot/renovate#8689] [Request Help]
**What Renovate type, platform and version are you using?**

Github private repo

**Describe the bug**

renovate-config-validator throws an error when [enabling experimental managers](https://docs.renovatebot.com/modules/manager/#enabling-experimental-managers)

**Relevant debug logs**

```
npm ERR! code 1
ERROR: renovate.json contains errors
       "errors": [
npm ERR! path /home/runner/work/project/Project
         {
           "depName": "Configuration Error",
           "message": "Invalid configuration option: experimental-manager"
         }
       ]
npm ERR! command failed
npm ERR! command sh -c renovate-config-validator
```

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [x] This is a really small bug, it does not need a reproduction (think small typo)
- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

I realized today that renovate now supports upgrading pre-commit plugins since [2020/11 with v23.86.0](https://github.com/renovatebot/renovate/releases/tag/23.86.0), but never received any PR to upgrade plugins. We haven't whitelisted the managers, and the [pre-commit doc](https://docs.renovatebot.com/modules/manager/pre-commit/) doesn't mention if it's a default manager or not. I guessed that it could be an experimental manager, and tried enabled them to see if that'll create pre-commit PRs.

**Comment by viceice** [comment]
this config option is not implemented 🤦‍♂️

 `pre-commit` manager should be enabled by default, please open a disucussion for `pre-commit` so we can workout the config problem. The `debug` logs will help for it too. Also a reproduction repo would help too.

**Comment by viceice** [comment]
`pre-commit` update should work out of the box. Please provice a reproduction repo and /or the debug logs.

[experimental-manager](https://docs.renovatebot.com/modules/manager/#enabling-experimental-managers) is only a placeholder for a real manager name.

**Reply by rarkins** [reply]
I have clarified the docs: https://github.com/renovatebot/renovate/commit/2d3b44401c9b6b72d1549d92e8852f1b3a417b69
